rfd/001-tcp-load-balancer-design-doc.md

```diff
@@ -58,16 +58,16 @@ For this project, a test client will be created and provided with a valid client
 Additionally, the requirement of over-the-wire encryption is not mentioned, so even though the authentication requires certs and keys, the data sent after authentication may or may not be encrypted depending on what is most convenient at the point of implementation.
 
 ### Rate Limiter
-This load balancer will implement a per-client connection rate limiter that tracks the number of client connections, and limits to n (configurable) active connections per client. It could track client identity based on attributes like IP address, and device identifier, but since this load balancer requires authentication (mTLS), we will identify unique clients based on a V5 UUID generated from the client's TLS certificate. This can be done leveraging the `crypto/tls` library, which makes peer certificates available on the `tls.Conn` struct returned from `tls.Client`. These can be turned into a consistent V5 uuid, to be used for `ClientID`, by creating a hash from the certificate, as shown below.
+This load balancer will implement a per-client connection rate limiter that tracks the number of client connections, and limits to n (configurable) active connections per client. It could track client identity based on attributes like IP address, and device identifier, but since this load balancer requires authentication (mTLS), we will identify unique clients based on a V5 UUID generated from the client's FQDN. This can be done leveraging the `crypto/tls` library, which makes peer certificate information available on the `tls.Conn` struct returned from `tls.Client`. These can be turned into a consistent V5 uuid, to be used for `ClientID`, by creating a hash from the FQDN, as shown below.
```

Comment:
This was an action item from last PR. I didn't get it added before approvals were given. It just changes the reference of what we are planning to hash for the clientID (cert -> FQDN).